### PR TITLE
[docker-compose] add how to get action log output to readme

### PR DESCRIPTION
**Pull request by alexkli**
This wasn't obvious to a newbie like me.

### CODE REVIEW

File: docker-compose/README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ The result of the invokation should be printed on the terminal:
 - OpenWhisk Controller - `~/tmp/openwhisk/controller/logs/`
 - OpenWhisk Invoker - `~/tmp/openwhisk/invoker/logs/`
 - `docker-compose` logs - `~/tmp/openwhisk/docker-compose.log`
+- Action output such as stdout or console.log(): `wsk -i activation logs <activationId>`
```
Comment (ddragosd) [inline docker-compose/README.md]: WDYT if we also cross-reference this tutorial : https://github.com/IBM-Bluemix/openwhisk-workshops/tree/master/bootcamp#start-your-engines , for more info, in addition to this quick command that you found useful.

Comment (alexkli) [reply docker-compose/README.md]: +1 seems like a helpful resource for me as well :-)

Comment (alexkli) [reply docker-compose/README.md]: done in section above

**Comment by ddragosd** [comment]
@alexkli feel free to merge with `master` branch to fix the build issues.

**Pull request merged**